<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpListener</class>
 <widget class="QWidget" name="TcpListener">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblConnection">
       <property name="text">
        <string>Choose TCP port to listen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinPort">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>64000</number>
       </property>
       <property name="value">
        <number>2424</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnect">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDisconnect">
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
